pkgs/applications/audio/surge-XT/default.nix

```diff
@@ -23,8 +23,8 @@ let
     src = fetchFromGitHub {
       owner = "lv2-porting-project";
       repo = "JUCE";
-      rev = "f7eb3b4681c1fb4735bcd388d7a5ed10ccd24aba";
-      sha256 = "sha256-jwWDB0XXaKkHJDNOSnhV4tDd+2lt61Qp8gFwXDuY63c=";
+      rev = "5106d9d77b892c22afcb9379c13982f270429e2e";
```

Comment:
Since this is technically a derivation as well, please update the date in the version above this as well.

Reply:
Sorry, I don't understand, please explain.

Reply:
https://github.com/NixOS/nixpkgs/blob/089ccb451a3cd8b141999c56edf8807095d57f7f/pkgs/applications/audio/surge-XT/default.nix#L20

needs to be bumped to `2021-12-11`

Reply:
Oh, of course!